src/NUnitFramework/framework/Constraints/SubstringConstraint.cs

```diff
@@ -30,6 +32,11 @@ namespace NUnit.Framework.Constraints
     public class SubstringConstraint : StringConstraint
     {
         /// <summary>
+        /// Indicates whether tests should be case-insensitive
```

Comment:
I think this message is just a little inaccurate because there's more going on than just cas-sensitivity, but you don't have to have XML docs on private members anyway.